docs/02-app/01-building-your-application/03-data-fetching/03-revalidating.mdx

```diff
@@ -15,7 +15,7 @@ There are two types of revalidation in Next.js:
 
 To revalidate cached data at a specific interval, you can use the `next.revalidate` option in `fetch()` to set the `cache` lifetime of a resource (in seconds).
 
-```jsx
+```js
```

Comment:
Should this have a file name like the others?

Reply:
I guess its valid in nearly every file so maybe its not necessary 🤷‍♂️

Reply:
Right. This is a generic example that could be used anywhere, which is why we don't put a filename. If we're creating a `Page` or `About` example, then we'll toss in a file name to orient people.